tensorflow/core/kernels/cudnn_rnn_ops.cc

```diff
@@ -650,20 +687,30 @@ Status DoForward(OpKernelContext* context, const RnnDescriptor& rnn_desc,
                  Tensor* output, Tensor* output_h, Tensor* output_c,
                  ScratchAllocator* reserve_space_allocator,
                  ScratchAllocator* workspace_allocator,
-                 ProfileResult* output_profile_result) {
+                 ProfileResult* output_profile_result,
+                 const Tensor* sequence_lengths=nullptr) {
```

Comment:
Move this above `output_profile_result` and remove the default value `nullptr` for uniform style.

Same for other places.